src/main/java/com/aws/greengrass/mqtt/bridge/clients/MQTTClient.java

```diff
@@ -242,6 +242,7 @@ private void updateSubscriptionsInternal() {
     private MqttConnectOptions getConnectionOptions() throws KeyStoreException {
         MqttConnectOptions connOpts = new MqttConnectOptions();
         connOpts.setCleanSession(true);
+        connOpts.setMaxInflight(Integer.MAX_VALUE);
```

Comment:
do we really want this?

What is the current default value?

Reply:
default value is 10

Reply:
What is the downside to increasing it?

Reply:
i think we shouldn't restrict inflight here, let the customer flood the system as much as they want--if we want a mechanism for backpressure / rate limiting i think it should be done in MessageBridge.

this would also make mqtt3 consistent with mqtt5 client, where we dont have inflight or flow control configured

Reply:
downside to increasing would be memory usage since messages would be sitting around waiting for their acknowledgement

Reply:
Inflight is configured automatically in mqtt 5 based on what the broker tells us. Having no limit sounds bad to me; paho will need to keep a data structure to know what has been acknowledged or not leading to huge memory usage. Set this to a more rational value like 1000 maybe.

Reply:
adjusted